### PR TITLE
Bngp 799 hmmp pre upload questions

### PR DESCRIPTION
**Pull request by shaundefra**
add Habitat Management Monitoring Plan pre-upload questions and skeleton cucumber reporter

### CODE REVIEW

File: src/conf/wdio.local.conf.js [hunk 1]
```diff
@@ -155,15 +157,22 @@ exports.config = {
...
+    //   "junit",
+    //   {
+    //     outputDir: "./",
+    //     outputFileFormat: function () {
+    //       // optional
+    //       //return `results-${options.cid}.${options.capabilities}.xml`;
+    //       return `TestReport.xml`;
+    //     },
+    //   },
+    // ],
```
Comment (timstone) [inline src/conf/wdio.local.conf.js:170]: can this be removed?

File: src/conf/wdio.local.conf.js [hunk 2]
```diff
@@ -120,14 +122,14 @@ exports.config = {
...
-      // },
-    ],
+    // [
+    // "selenium-standalone",
+    // {
+    //   logPath: "logs",
+    //   installArgs: { drivers: { chrome: { version: "103.0.5060.53" } } },
+    //   args: { drivers: { chrome: { version: "103.0.5060.53" } } },
+    // },
+    // ],
```
Comment (timstone) [inline src/conf/wdio.local.conf.js:132]: can this be removed?

File: src/features/hmmp-pre-upload-questions.feature [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    I need to answer a set of questions about the habitat management of my site
+    So that I can provide required information
+
+    Scenario: I can add a valid habitat enhancement works start date
+        Given I am on the "habitat-works-start-date" page
+        When I enter a valid startdate of "12/10/2022"
+        Then I should be on the "monitoring-start-date" page
+
+    Scenario: I can add a valid 30 year monitoring start date
+        # Given I have added a valid "habitat-works-start-date"
```
Comment (timstone) [inline src/features/hmmp-pre-upload-questions.feature:14]: can this be removed?

File: src/features/hmmp-pre-upload-questions.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        Given I am on the "monitoring-start-date" page
+        Then I should be able to enter a valid 30 year monitoring startdate of "13/10/2022"
+# Todo: assert against next page.  CYA page for hmmp not built yet,
+# currently just asserting against the input and that it does not error
```
Comment (timstone) [inline src/features/hmmp-pre-upload-questions.feature:18]: Couldn't you add these steps and just set the step definition to pending?

File: src/conf/wdio.local.conf.js (hunk 1 above)
Comment (shaundefra) [reply src/conf/wdio.local.conf.js]: updated

**Review by timstone — APPROVED** [review]

**Pull request merged**